fe/fe-core/src/main/java/com/starrocks/qe/StmtExecutor.java

```diff
@@ -1454,6 +1478,55 @@ private void handleUpdateFailPointStatusStmt() throws Exception {
         executor.execute();
     }
 
+    private void handleDeallocateStmt() throws Exception {
+        DeallocateStmt deallocateStmt = (DeallocateStmt) parsedStmt;
+        String stmtName = deallocateStmt.getStmtName();
+        if (context.getPreparedStmt(stmtName) == null) {
+            throw new Exception("PrepareStatement `" + stmtName + "` not exist");
+        }
+        context.removePreparedStmt(stmtName);
+        context.getState().setOk();
+    }
+
+    private void handlePrepareStmt(ExecPlan execPlan) throws Exception {
+        PrepareStmt prepareStmt = (PrepareStmt) parsedStmt;
+        boolean isBinaryRowFormat = context.getCommand() == MysqlCommand.COM_STMT_PREPARE;
```

Comment:
must true?

Reply:
no， only using prepare statement in jdbc client  will send cmd COM_STMT_PREPARE. if use mysql client , won't need binary format. its text format

Reply:
so mysql client don't need any reply if use mysql client ?

Reply:
yes